### ISSUE TITLE
False-positive no-multi-spaces warnings on javadoc comments

### ISSUE THREAD

**Issue by carstenhag** [pinterest/ktlint#706]
## Expected Behavior
IMO, the multiple spaces aligning the parameters and their explanation help by making them easier to read.

## Observed Behavior
In dozens of files this warning is shown. It is unreasonable to disable it per file. If I disable it globally though, I will miss things like `SomeInterface  {`, which has too many spaces.

## Steps to Reproduce

```kt
    /**
     * Gets Blabla from user.
     *
     * @param blabls12      1234
     * @param blabla123     6789
     * @param blabla        5678
     * @param longparam345  4567890
     * @param userId        567890
     * @return the user profile
     */
```
## Your Environment

* Version of ktlint used: Probably newest ktlint
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): Gradle Plugin 9.2.1

**Comment by carstenhag** [comment]
Thanks a lot for implementing this :)
